src/pineko/evolve.py

```diff
@@ -245,10 +245,6 @@ def write_operator_card(pineappl_grid, default_card, card_path, tcard):
     kv = pineappl_grid.key_values()
     conv_type_1, conv_type_2 = get_grid_convolution_type(kv)
 
-    # fragmentation function grid?
-    if "timelike" in kv:
```

Comment:
Actually the previous implementation was not even working cause the "True" business